### DISCUSSION TITLE
Cannot pin shortcuts to Open Shell Start

### DISCUSSION THREAD

**Discussion by milleron387** [Open-Shell/Open-Shell-Menu#1807] [General]
IIRC, I must have once had the ability to pin items to the Open Shell Start Menu. Currently, in Windows 11 22H2, the "Pin to Start" option appears on right-clicking a shortcut, but it pins the item only to the Windows Start, not to the Open Shell Start Menu. I cannot find an option in Open Shell settings to designate where that pin-to-Start process should occur. In case there might be a maximum number of items pinned to Open Shell Start, I unpinned one, but I still cannot pin the new one. How is that done?

**Comment by ge0rdi** [comment]
You should use `Pin to Start menu (Open-Shell)` context menu item.
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/2453094/0023a0cc-c5a0-43c4-8e8c-310ace8b3995)

**Reply by milleron387** [reply]
That's just it. There is **_no such item_** on my context menu. It reads simply "Pin to Start menu" period -- no mention of "Open-Shell."
I can find nothing in Open-Shell Settings that has anything to do with that omission.

**Reply by ge0rdi** [reply]
Check the `Disable the Pin shell extension` setting.
Do you have it enabled?

**Comment by milleron387** [comment]
I did not have it enabled. I tried enabling, clicking "Apply," and then disabling it again with no result
I was on version 4.4.190 and updated to 4.4.191 (which was problematic as the setup could not close Windows Explorer causing a mostly blank screen with loss of the desktop, Task Bar and desktop icons but setup did finally complete after a couple of attempts). There _**still**_ is no context-menu entry for "Pin to Start menu (Open-Shell)"
This used to work flawlessly. I wonder if it got messed up by upgrading from Windows 10 to Windows 11 a few months ago.

**Comment by milleron387** [comment]
I completely uninstalled Open-Shell via Revo Uninstaller and then reinstalled it. As opposed to the update process, it did not require "closing Windows Explorer." Version 4.4.191 installed instantly. "_Disable the Pin Shell extension_" is still UNchecked, and there is still no "Pin to Start menu (Open-Shell)" in the context menu for desktop icons, _only_ the "_Pin to Start_" item.

I'm starting to wonder if this is a problem caused by a Windows 11 update rather than a problem wtih Open-Shell.

**Reply by ge0rdi** [reply]
Could it be that you are referring to _modern_ Win11 context menu that shows just few items by default?
Try to click `Show more options` (usually last item). That should show _classic_ context menu with all the items (just like in Win10).

**Reply by milleron387** [reply]
That was exactly what the problem was. I'm very well aware of the need to check the "More options" in Win 11. Why it didn't occur to me to do it this time I'll never know. But thanks for your patience and help. I'm good to go. (For a couple of days there, I thought I'd have to pay Stardock, but if I had, I'd almost certainly have run into the same problem. 🥴)
